ha/core/controllers/pcs/node_controller.py

```diff
@@ -367,7 +368,15 @@ def stop(self, node_id: str, timeout: int = -1, **op_kwargs) -> dict:
                 # Stop services on node except sspl-ll
```

Comment:
Please correct this comment

Reply:
what correction is needed here ? 
We need to stop everything except sspl at this point

Reply:
Never mind